kube-core/src/request.rs

```diff
@@ -319,20 +312,13 @@ impl Request {
             qp.append_pair("continue", continue_token);
         }
 
-        match &lp.version_match {
-            VersionMatch::MostRecent => (),
-            VersionMatch::Any => {
-                qp.append_pair("resourceVersion", "0");
-                qp.append_pair("resourceVersionMatch", "NotOlderThan");
-            }
-            VersionMatch::NotOlderThan(resource_version) => {
-                qp.append_pair("resourceVersion", resource_version.as_str());
-                qp.append_pair("resourceVersionMatch", "NotOlderThan");
-            }
-            VersionMatch::Exact(resource_version) => {
-                qp.append_pair("resourceVersion", resource_version.as_str());
-                qp.append_pair("resourceVersionMatch", "Exact");
-            }
+        if let Some(rv) = &lp.resource_version {
```

Comment:
Isn't this.. the same as above?

Reply:
yeah, `list` vs `list_metadata`. i think there's a bigger refactor lurking in here to do a `From<ListParams> for QueryStringType` - but didn't want to overdo it.

Reply:
Fair.